roles/matrix-cactus-comments/defaults/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Cactus Comments is a federated comment system built on Matrix
+
+matrix_cactus_comments_enabled: true
+matrix_cactus_comments_container_image_self_build: false
+matrix_cactus_comments_docker_repo: "https://gitlab.com/cactus-comments/cactus-appservice.git"
+matrix_cactus_comments_docker_repo_version: "{{ matrix_cactus_comments_version if matrix_cactus_comments_version != 'latest' else 'main' }}"
```

Comment:
When self-building is enabled (it may be manually enabled or it may automatically get enabled if you're on an unsupported CPU architecture), then the image will be built from source from that git repository. When the Docker version is set to `latest`, we'll automatically build the `main` git repository branch. If it's pinned to some version (e.g. `v1.2.3`), we'll automatically pull the branch that goes by the same name (`v1.2.3`).

It seems fine.